settings.gradle

```diff
@@ -11,3 +15,5 @@ include ":tool-googleplay"
 include ":tool-appgallery"
 include ":tool-thimble"
 include ":tool-timber"
+
+gradle.startParameter.excludedTaskNames.add(":buildSrc:testClasses")
```

Comment:
Newest AS has this problem where you can't compile anything. AS is blaming Gradle and vice versa. We have commited this on our project as we are waiting for it to be fixed, so it should be ok here? [OPTIONAl]

Reply:
So this solves the issue? Test classes won't be compiled?

Reply:
No, test classes are still compiled 😄 .
https://issuetracker.google.com/issues/315023802

Reply:
RIP